Dockerfile

```diff
@@ -1,16 +1,20 @@
 FROM node:alpine
 
-ENV NODE_ENV production
-ENV HOST 0.0.0.0
-ENV PORT 80
-
 # ENV RAZZLE_CUSTOM_VARIABLE x
 
-# Bundle APP files
-COPY build .
-COPY package.json .
-COPY build/public public
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# install app dependencies
+COPY package.json /usr/src/app/
+RUN yarn install
+
+# bundle app source
+COPY . /usr/src/app
+
```

Comment:
I dont think we need to specify the full absolute path after defining the Workdir step. you can just use `.`